assets/css/styles.css

```diff
@@ -557,3 +558,126 @@ button {
     background-size: 100%;
   }
 }
+
+#pop-section {
+  position: fixed;
+  height: 100vh;
+  display: none;
+  border-radius: 8px;
+  width: 100%;
+  top: 0;
+  background: #fff;
+  flex-direction: column;
+  align-items: center;
+}
+
+#pop-section .close-wrapper {
+  width: 94%;
+  display: flex;
+  flex-direction: row;
+  justify-content: end;
+  margin-top: 1em;
+}
```

Comment:
- [ ] [REQUIRED] Please fix layout and margin, padding errors on your desktop version modal like a [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)!

![image](https://user-images.githubusercontent.com/63915024/189193235-78fea027-6eed-4cfc-8009-b23664b8db02.png)

Reply:
Hello @mukhammadrizooff , Thanks for the review. However, I see that from the figma design, the close button is just above the image. Please advise